### ISSUE TITLE
iOS error - Value for (key = topicsDialogMinimumSessions) should not be null

### ISSUE THREAD

**Issue by vladsyd19** [cleverpush/cleverpush-react-native-sdk#6]
Hi, I faced with app crash due to error on iOS:
`Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Value for (key = topicsDialogMinimumSessions) should not be null'
terminating with uncaught exception of type NSException`
It happens after allowing send notifications permissions. 
Also before an error I see in console:
```
[CleverPush] INFO: syncSubscription request data:{
    apnsToken = 'my apnsToken';
    appVersion = "1.0.7";
    browserType = SDK;
    browserVersion = "1.22.5";
    country = UA;
    language = en;
    platformName = iOS;
    platformVersion = "16.1.1";
    timezone = "Europe/Kiev";
    topics =     (
    );
    topicsVersion = 1;
} id:(null)
```
I made all of the steps from React Native SDK setup iOS except optional 8. Add AppGroup step
Also called `CleverPush.init` with my channel id by app start. 

In React Native SDK setup I didn't found any information about how to get device token for iOS, so I added next lines to `AppDelegate.m` by myself:
`#import <CleverPush/CleverPush.h>`
...
` - (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken
 {
   [CleverPush didRegisterForRemoteNotifications:application deviceToken:deviceToken];
 }
  - (void)application:(UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error
 {
   [CleverPush handleDidFailRegisterForRemoteNotification:error];
 }`
 
 package.json:
 "cleverpush-react-native": "^1.4.9" (1.4.10 - tried both),
 "react-native": "0.64.3"
 
 What I tried? We bought a CleverPush paid subscription and I have play with value in Channel -> Topics -> Settings -> Min. sessions , tried to change it to 0 and 1, but nothing helped.
 
 By the way, my device is successfully registered to get notifications, I see it at CleverPush my channel subscription tab
 
 I really appreciate your help!

**Comment by gbhrdt** [comment]
Hi @vladsyd19  thanks for your report!

1) This is not needed, as we automatically do this by Swizzling some classes in iOS:
 
`#import <CleverPush/CleverPush.h>`
...
` - (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken
 {
   [CleverPush didRegisterForRemoteNotifications:application deviceToken:deviceToken];
 }
  - (void)application:(UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error
 {
   [CleverPush handleDidFailRegisterForRemoteNotification:error];
 }`
 
2) Currently, we are unable to replicate this – are you maybe able to provide us with a small replication project? That would be awesome!

**Comment by gbhrdt** [comment]
Hi @vladsyd19 thanks for that replication project! We identified that some other Pod dependency which is used in your project is also creating a custom Objective-C category for the `NSDictionary` class, which has the same method names as our `SafeExpectations` category. We will deploy a fix for this by renaming our methods asap, I will keep you updated here.

**Comment by gbhrdt** [comment]
@vladsyd19 We just released Version 1.4.11 which should fix your issue, thanks for your patience.

**Comment by vladsyd19** [comment]
It works! Thank you
